server/registry/server.go

```diff
@@ -95,6 +95,14 @@ func (s *RegistryServer) getStorageClient(ctx context.Context) (*storage.Client,
 	return s.storageClient, nil
 }
 
+func (s *RegistryServer) runWithTransaction(ctx context.Context, fn func(ctx context.Context, db *storage.Client) error) error {
```

Comment:
nit: Personally, I would prefer `runInTransaction()` or `asTransaction()`.

Reply:
Thanks, I'll change this to "runInTransaction"

Reply:
Done